src/quo2/components/selectors/react/view.cljs

```diff
@@ -0,0 +1,25 @@
+(ns quo2.components.selectors.react.view
+  (:require [quo2.components.selectors.react-selector.view :as react-selector]
+            [quo2.components.selectors.react-selector-add.view :as react-selector-add]
+            [quo2.components.selectors.react.style :as style]
+            [react-native.core :as rn]))
+
+(defn view
+  [{:keys [reactions on-press on-long-press add-reaction? on-press-add pinned? container-style]}]
+  [rn/view {:style (merge container-style style/container)}
```

Comment:
I think this should be swapped in order as `container-style` is an overwrite. i.e
`(merge style/container container-style)`